src/inference/dev_api/cpp_interfaces/interface/ie_internal_plugin_config.hpp

```diff
@@ -108,6 +108,14 @@ DECLARE_CONFIG_KEY(CONFIG_DEVICE_ID);
  * @brief enable hyper thread
  */
 DECLARE_CONFIG_KEY(ENABLE_HYPER_THREAD);
+
+/**
+ * @brief Defines if MHA ops can be tokenized in Snippets
+ *        Softmax, Transpose should be tokenized in Snippets only in tests and in MHA pattern
+ * @ingroup ie_dev_api_plugin_api
+ */
+DECLARE_CONFIG_KEY(SNIPPETS_MHA_OPS_TOKENIZATION_ENABLE);
+
```

Comment:
It's a good idea, but it seems to me that it could not be done without major refactoring of `QueryNetwork`, `LoadExeNetworkImpl` and `SubgraphBaseTest` (all of them take the config map that is expected to define the plugin behavior). If you are certain that we can do it without a significant impact on non-snippets code (and we will still be able to set the config from the tests), let me know please, we can change it.